src/index.test.ts

```diff
@@ -63,6 +64,77 @@ describe("voices", () => {
 });
 
 describe("tts.websocket", () => {
+  let phonicWebSocket: PhonicWebSocket;
+
+  /**
+   * Promise that resolves with all messages received from the websocket after a period of no messages. Can be awaited
+   * multiple times.
+   */
+  let allMessagesReceived: Promise<PhonicWebSocketResponseMessage[]> | null =
+    null;
+  const maxIdleTime = 5000; // If we don't receive messages for 5 seconds, we consider that we received all of them.
+
+  const safeGet = (
+    messages: PhonicWebSocketResponseMessage[] | null,
```

Comment:
How can it be null?

Reply:
It shouldn't ever be null, but typescript thinks it can because`allMessagesReceived` is typed as
```typescript
let allMessagesReceived: Promise<PhonicWebSocketResponseMessage[]> | null = null;
```

I fixed this by creating a separate helper function `getMessages` that asserts the messages are not-null before returning.

Reply:
I don't see why would `allMessagesReceived === null` imply that `messages` can be null.
If messages is typed as `PhonicWebSocketResponseMessage[]`, initialized as `[]`, and when you push an item it's always a `PhonicWebSocketResponseMessage`, TypeScript shouldn't complain. Is it not the case?

Reply:
My question is simple: 
Why `messages` can't be typed as `PhonicWebSocketResponseMessage[]` (without the null)?

Reply:
> If messages is typed as PhonicWebSocketResponseMessage[], initialized as [], and when you push an item it's always a PhonicWebSocketResponseMessage, TypeScript shouldn't complain. Is it not the case?

You're correct here about messages -- I believe what typescript is confused about is the fact that the promise (which returns messages) itself could be null, and `await null` will return null.

Reply:
> My question is simple: Why `messages` can't be typed as `PhonicWebSocketResponseMessage[]` (without the null)?

This is what happens currently when I try to type it without the null:
![image](https://github.com/user-attachments/assets/c18789a8-0749-4c95-bb0c-de068ed58ab8)

Reply:
The current implementation of `safeGet` already types `messages` as `PhonicWebSocketResponseMessage[]`:

https://github.com/Phonic-Co/phonic-node/blob/02d660f3589117eba48c307ad2cd21dccbdaf9c2/src/index.test.ts#L84

The way that I achieved it was adding the `getMessages` function.

Reply:
Alright, now I see what you did. This is great!